src/celeritas/optical/gen/PrimaryGenerator.hh

```diff
@@ -39,7 +37,7 @@ class PrimaryGenerator
     //// DATA ////
 
     PrimaryDistributionData const& data_;
-    IsotropicDistribution<real_type> sample_angle_;
+    IsotropicDistribution<real_type> sample_polarization_;
```

Comment:
Make sure to note for our separable user distribution that the sampled light is unpolarized.